clients/unshaded/src/main/java/tachyon/client/netty/NettyRemoteBlockReader.java

```diff
@@ -88,4 +93,18 @@ public ByteBuffer readRemoteBlock(String host, int port, long blockId, long offs
       throw new IOException(e);
     }
   }
+
+  /**
+   * Release the underlying buffer of previous/current read response.
+   */
+  @Override
+  public void close() throws IOException {
```

Comment:
Just the response. @gpang and I discussed on whether it should be called `close()` or `closeResponse` and decided the former as externally it makes more sense.